### PR TITLE
Add issuer name tests.

### PR DESCRIPTION
**Pull request by aljones15**
Adds a test for "If present, the value of the issuer.name property MUST be a string or a language value object as described in 10.1 Language and Base Direction."

1. create an invalid "value object" with an extra prop like "url"
2. create a value object array with multiple languages
3. add a test with `@direction`

### CODE REVIEW

**Review by BigBlueHat — CHANGES_REQUESTED** [review]
@aljones15 good start. I think key to these specific tests will be negative versions. The "value object" for example, MUST NOT have any additional properties. So the following would be an invalid value for `name`:
```json
{
      "@value":"ExampleIssuer",
      "@language": "en",
      "url": "http://example.com/issuer" 
    }
```

And arrays of value objects are also valid--to support multiple languages.

We should also get an `@direction` example tested in their also.

**Review by BigBlueHat — APPROVED** [review]
Excellent. Thank you!

**Review by JSAssassin — APPROVED** [review]

File: tests/10-vcdm2.js [hunk 1]
```diff
@@ -317,6 +317,19 @@ describe('Verifiable Credentials Data Model v2.0', function() {
         await assert.rejects(
           issue(require('./input/credential-issuer-object-no-id-fail.json')));
       });
+      it2('If present, the value of the "issuer.name" property MUST be a ' +
+      'string or a language value object as described in 10.1 Language and' +
+      ' Base Direction.', async function() {
```
[the remark this answers is not in the document]
Comment (aljones15) [reply tests/10-vcdm2.js:322]: I'll take this, but it probably breaks the linter.

**Comment by JSAssassin** [comment]
Also need to add a changelog entry for the test like in the other PR.

**Pull request merged**